mobile/src/main/java/org/openhab/habdroid/util/Constants.java

```diff
@@ -22,7 +22,7 @@
     public static final String PREFERENCE_ANIMATION         = "default_openhab_animation";
     public static final String PREFERENCE_DEMOMODE          = "default_openhab_demomode";
     public static final String PREFERENCE_FULLSCREEN        = "default_openhab_fullscreen";
-    public static final String PREFERENCE_TONE              = "default_openhab_alertringtone";
```

Comment:
I think you removed that by accident

Reply:
You're right, I already found that and fixed it in the next commit, however, I forgot to push it, sorry! :)